public/stylesheets/main.css

```diff
@@ -654,27 +634,6 @@ p#conditional-text {
     margin-bottom: 20px;
 }
 
-#press-container, #collaborators-container {
```

Comment:
You deleted the CSS for the `press-container` element here, but it was actually being shared between `press-container` and `collaborators-container`. We are keeping the collaborators section, so this messes up the formatting of that section of the landing page. So you should just delete the reference to press container here instead of removing the CSS as a whole.